apollo-compiler/src/test/graphql/com/example/root_query_inline_fragment/TestOperation.graphql

```diff
@@ -0,0 +1,26 @@
+query TestQuery {
+  ... on Query {
+    __typename
+    hero {
+      ...heroFragment
+    }
+    droid(id: 1) {
+      ...droidFragment
+    }
+    human(id: 1) {
+      ... on Human {
+       name
+       homePlanet
+      }
+    }
+  }
+}
+
+fragment heroFragment on Character {
+  name
+}
+
+fragment droidFragment on Droid {
+  name
+  primaryFunction
+}
```

Comment:
should we omit named fragment to make this test fixture more simple? As I understood we testing just a case when inline fragment is used as root.

Reply:
Sure! Removed in 6fbb8af.